addons/analytic/views/analytic_line_views.xml

```diff
@@ -121,31 +121,25 @@
         <field name="model">account.analytic.line</field>
         <field name="arch" type="xml">
             <kanban class="o_kanban_mobile">
-                <field name="date"/>
-                <field name="name"/>
+                <!-- Doubt : Why? If we remove account_id from here,then Departments and Internal fields value will be displayed in the Kanban view. -->
```

Comment:
wow you're right, there's a weird (wrong?) processing of the view in python. If they're not there, they are added but as "optional", so I assume this is a list view logic that applies to kanban.

@william-andre could you clarify this?

Reply:
Anyway, I suggest something like this for this arch:
```xml
<kanban class="o_kanban_mobile">
    <field name="account_id"/>
    <field name="currency_id"/>
    <templates>
        <t t-name="kanban-card">
            <div class="row">
                <field class="col-7 fw-bold fs-5" name="name"/>
                <field class="col-5 text-end fw-bold fs-5" name="date"/>
            </div>
            <div class="row">
                <field class="col-7 text-muted" name="account_id"/>
                <field class="col-5 justify-content-end" name="amount" widget="monetary"/>
            </div>
        </t>
    </templates>
</kanban>
```
And I think that a flex row display would be even better that the row/col-7/col-5

Reply:
I actually don't know what we want to display in the kanban for the accounts...
Basically, `account_id` is just the "first" analytic plan and every time it is used we want to have all of the other plans as well, in all the views.
Kanban might be different because the space is more limited, but there is no reason to display `account_id` without the other plans.